code/modules/assembly/igniter.dm

```diff
@@ -24,6 +24,9 @@
 	var/turf/location = get_turf(loc)
 	if(location)
 		location.hotspot_expose(1000,1000)
+		for(var/T in location.contents)
+			var/atom/A = T
+			A.fire_act()

```

Comment:
It ignites but it doesn't hold any fuel. It should not light things on fire without fuel.